docs/site/output.md

```diff
@@ -136,6 +136,10 @@ In full stack projects, you will probably use the window as a REPL for both `clj
 
 ![Toggle CLJC](images/howto/cljc-toggle-button.png)
 
+## REPL process output (stdout and stderr)
+
+When evaluating code, `stdout`, and `stderr` stemming from `(println ...)` et al is captured by Calva and sent to the Output window. By default this includes output from threads spawned by the evaluated code as well as a lot of other output from the repl process. You can opt out of the “this includes” behaviour with the `calva.redirectServerOutputToRepl` setting.
```

Comment:
```suggestion
When evaluating code, `stdout`, and `stderr` stemming from `(println ...)` et al is captured by Calva and sent to the Output window. By default this includes output from threads spawned by the evaluated code as well as a lot of other output from the repl process. You can opt out of this behavior by setting the `calva.redirectServerOutputToRepl` setting to `false`.
```

I think it's fine to just say "this behavior" as it implies the previously stated behavior. I also think it's clearer to say what value they should give that setting to opt-out.

Reply:
Maybe the whole section is a bit unclear. What I am trying to say is that the setting only affects part of the output. That's why I tried to qualify which behaviour. I'm merging your suggestion, and will look at the section again to see if I can clarify what it all is about.

Reply:
I've pushed an updated version of the copy. Please have a look and see if you think it's an improvement.

Reply:
Looks good to me!